packages/bz3.rb

```diff
@@ -3,7 +3,7 @@
 class Bz3 < Package
   description 'bzip3 is a better and stronger spiritual successor to bzip2.'
   homepage 'https://github.com/kspalaiologos/bzip3'
-  @_ver = '1.1.6'
+  @_ver = '1.1.8'
   version @_ver
   license 'LGPL-3'
   compatibility 'all'
```

Comment:
Why don't we have binaries?  Is the build quick?

Reply:
my containers are malfunctioning right now, i'm not sure what's wrong.